.circleci/config.yml

```diff
@@ -2475,32 +1717,7 @@ workflows:
      #       - benchmarks-loaded-agent-single-50mb-log-file-with-500k-lines-py-27
      #       - benchmarks-loaded-agent-single-50mb-log-file-with-500k-lines-py-35
      #     <<: *benchmarks_master_and_release_only
- weekly-circle-ci-usage-report:
-   triggers:
-     - schedule:
-         cron: "0 0 * * 0"
-         filters:
-           branches:
-             only:
-               - master
-   jobs:
-     - send-circle-ci-usage-report
 
- hourly-windows-ami-tests:
-   triggers:
-     - schedule:
-         cron: "0 * * * *"
-         filters:
-           branches:
-             only:
-               - win_py30_changes
-   jobs:
-     - ami-tests-development-windows:
-         context: scalyr-agent
-         requires:
-            - build-windows-package
-     - build-windows-package
-     - build-linux-packages-and-installer-script
```

Comment:
There is no equivalent for these yet.

Reply:
Yeah, that was actually a left over build from the past which we forgot to remove it (it was only triggered for ``win_py30_changes`` branch).

We have another target which basically does the same for ``master`` branch.